WebServices/AzureNotificationHub/NotificationHubSample/NotificationHubSample.Android/FirebaseService.cs

```diff
@@ -34,6 +37,13 @@ public override void OnMessageReceived(RemoteMessage message)
             SendMessageToMainPage(messageBody);
         }
 
+        public override void OnNewToken(string token)
```

Comment:
The old Firebase implementation required an additional class that subclassed `FirebaseInstanceIdService` (see deleted file in this PR). That class was responsible for registering with Firebase and obtaining a token. Google updated this so that this `FirebaseMessagingService` class is now responsible for implementing `OnNewToken`. The contents of the deleted class essentially moved into this class.

One consequence of this is it is no longer possible to retrieve the token using `FirebaseInstanceId.Instance.Token` (obsoleted). It may be possible to request it by calling into native Android methods but the easiest way is to just save the token locally in this method.